music_assistant/server/providers/snapcast/__init__.py

```diff
@@ -318,10 +320,8 @@ async def unload(self) -> None:
             player_id = self._get_ma_id(snap_client_id)
             await self.cmd_stop(player_id)
         self._snapserver.stop()
-        if self._snapserver_runner and not self._snapserver_runner.done():
-            self._snapserver_runner.cancel()
-        await asyncio.sleep(10)  # prevent race conditions when reloading
-        self._snapserver_started.clear()
+        await asyncio.sleep(10)
```

Comment:
what's up with the sleep here ? That is not so nice as it will delay reload of the provider and server shutdown even.

Reply:
Yes, my bad